components/mpas-ocean/src/Registry.xml

```diff
@@ -284,49 +284,48 @@
 					description="Characteristic length scale, usually the smallest dx in the mesh"
 					possible_values="any positive real"
 		/>
-		<nml_option name="config_Leith_visc2_max" type="real" default_value="2.5e3" units="m^2 s^{-1}"
+		<nml_option name="config_Leith_visc2_max" type="real" default_value="2.5e3" units="m^2 s^-1"
 					description="Upper bound on the allowable value of Leith-computed viscosity"
 					possible_values="any positive real"
 		/>
 	</nml_record>
 	<nml_record name="Redi_isopycnal_mixing" mode="forward">
-		<nml_option name="config_use_Redi" type="logical" default_value=".false." units="unitless"
+		<nml_option name="config_use_Redi" type="logical" default_value=".false."
 					description="If true, Redi isopycnal mixing is turned on"
 					possible_values=".true. or .false."
 		/>
-		<nml_option name="config_Redi_closure" type="character" default_value="constant" units="unitless"
+		<nml_option name="config_Redi_closure" type="character" default_value="constant"
 					description="Control what type of function is used for Redi $\kappa$. For 'equalGM', RediKappa is set to gmBolusKappa, so picks up the closure used by GM. Note that equalGM should only be used with 2D GM schemes (e.g. config_GM_closure=constant or Visbeck), not with EdenGreatbatch."
 					possible_values="'constant', 'equalGM', 'data'"
-		/>
-		<nml_option name="config_Redi_constant_kappa" type="real" default_value="600.0" units="m^2 s^{-1}"
+		<nml_option name="config_Redi_constant_kappa" type="real" default_value="600.0" units="m^2 s^-1"
```

Comment:
Oops, the `/>` needs to be added back for `config_Redi_closure`.  This is causing:
```
mpas_ocn_gm.F:782:89:

  782 | iKappa(iEdge) = RediHorizontalTaper(iEdge) * config_Redi_constant_kappa
      |                                                                       1

Error: Symbol 'config_redi_constant_kappa' at (1) has no IMPLICIT type
```